mon-pix/app/templates/certifications/start.hbs

```diff
@@ -7,7 +7,12 @@
 
         {{#if model.isCertifiable}}
           <div class="rounded-panel rounded-panel--strong rounded-panel--over-background-banner certification-start-page__panel">
-            <Stepper @steps={{array "certification-stepper-start"}}/>
+            {{#if this.showCongratulationsBanner }}
```

Comment:
Je verrais bien une variable inverse de `showCongratulationsBanner ` qui s'appellerait `closeCongratulationsBanner` et ça donnerait : `{{#unless this.closeCongratulationsBanner }}`

Reply:
vu avec jonas, no `unless` !